src/containers/homepage/survey-list.component.tsx

```diff
@@ -62,6 +90,65 @@ export class SurveyListComponent extends Component<SurveyListComponentProps> {
                 ),
             },
         ];
-        return <Table dataSource={items} columns={columns} bordered />;
+
+        return (
+            <>
+                <Row type="flex" justify="space-between">
```

Comment:
I will fix this... later.

Reply:
Thank you. Table should have a `min-width` as well.

Reply:
Most important thing to check for here is behaviour when the table doesn't fit on narrow screens

Reply:
Do we support narrow screens? At this point I would set a `min-width` of 1024 pixels and say screw it to any device less wide than that.

Reply:
Sure, works for me